app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -448,17 +445,17 @@ private boolean scanPokemonScreen() {
     private boolean infoLayoutArcPointerVisible = false;
 
     private void showInfoLayoutArcPointer() {
-        if (!infoLayoutArcPointerVisible && arcPointer != null && infoLayout != null) {
+        if (!infoLayoutArcPointerVisible && arcPointer != null && inputsLayout != null) {
```

Comment:
Eventually (probably in a later commit) we might want this flag to go (not sure)? (Though IIRC we added it to prevent ` windowManager.removeView(arcPointer);` when `arcPointer` isn't there, because that triggered a crash).